src/planwise/client/effects.cljs

```diff
@@ -102,8 +102,7 @@
                       (dissoc :key)
                       request->xhrio-options
                       ajax/ajax-request)]
-        ;FIXME: Issue #456
-        (when key (swap! rf-db/app-db assoc-in key xhrio))))))
+        (when key (swap! rf-db/app-db assoc-in [key] xhrio))))))
```

Comment:
This still modifies the re-frame database directly, which we should be trying to avoid. Instead use a separate atom local to this namespace.